pkg/ccl/streamingccl/streamingest/stream_ingestion_frontier_processor.go

```diff
@@ -329,6 +335,9 @@ func (sf *streamIngestionFrontier) maybeUpdateProgress() error {
 		streamProgress := progress.Details.(*jobspb.Progress_StreamIngest).StreamIngest
 		streamProgress.Checkpoint.ResolvedSpans = frontierResolvedSpans
 
+		// Initialize retained time to cutover time to compare with persisted replicated time later.
+		sf.retainedTime = streamProgress.CutoverTime
```

Comment:
nit: it seems clearer to set `streamProgress.CutoverTime` to a local variabe `cutoverTime`, and then set `retainedTime` in a single place below.

Reply:
done